changelog.txt

```diff
@@ -1,3 +1,10 @@
+Version 3.17.4
```

Comment:
Should be 3.18, really as we have additions to the public API.

Reply:
done.